content/workshops/openshift_4_101/lab5-rollbacks.md

```diff
@@ -177,30 +201,21 @@ $ oc get pods -w
 {{% panel "Web Console Steps" %}}
 
 <blockquote>
-Click on "Applications" and then click on "Deployments"
-</blockquote>
-This is going to show basic details for all deployment configurations in this project
-
-<blockquote>
-Click the "dc-metro-map" deployment config
-</blockquote>
-Toward the bottom of the screen you will see a table of deployments using this deployment config
-<img src="../images/ocp-lab-rollbacks-deploymentconfigsummary1.png" width="900"><br/>
-
-<blockquote>
-In the Deployments table click the #1
+Click on "Builds" and then click on "Builds"
+<br/>
+This is going to show basic details for all builds, for the dc-metro-map application.
 </blockquote>
-<img src="../images/ocp-lab-rollbacks-deploymentconfig.png" width="600"><br/>
+<img src="../images/ocp-lab-rollbacks-builds-builds.png" width="900"><br/>
 
 <blockquote>
-Click the "Rollback button", accept defaults, and click "Rollback" again
+Click the "dc-metro-map" build that you want to roll back to. For the purposes of this lab, click dc-metro-map-1.
+<br/>
+Click on 'Actions', and then 'Rebuild', from the menu, in the upper right corner of the window.
```

Comment:
Click 'Actions.' Then, click 'Rebuild' from the menu in the upper right.
Note: These menus should be labeled so that less effort needs to be given to the directions. Like, "Click Rebuild from the XYZ Menu"